spec/rails_helper.rb

```diff
@@ -2,16 +2,20 @@
 require 'spec_helper'
 require File.expand_path("../../config/environment", __FILE__)
 require 'rspec/rails'
+require 'webmock/rspec'
+
+Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 
 ActiveRecord::Migration.maintain_test_schema!
 
 RSpec.configure do |config|
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
 
-  config.use_transactional_fixtures = true
+  config.use_transactional_fixtures = false
```

Comment:
this was screwing up the postgres json query for some reason... http://stackoverflow.com/questions/21138207/activerecordstatementinvalid-pg-infailedsqltransaction